CRM/Admin/Form/Setting/Localization.php

```diff
@@ -354,11 +354,15 @@ public static function onChangeDefaultCurrency($oldCurrency, $newCurrency, $meta
    * @return array
    */
   public static function getDefaultLanguageOptions() {
-    return [
+    $availableOptions = [
       '*default*' => ts('Use default site language'),
       'undefined' => ts('Leave undefined'),
       'current_site_language' => ts('Use language in use at the time'),
     ];
+    // Get a list of all available languages
+    $availableLanguages = CRM_Core_I18n::languages();
+    // Merge those 2 arrays
+    return array_merge($availableOptions, $availableLanguages);
```

Comment:
The comments are a bit redundant. We can assume that people can read PHP :-)
Maybe document something like "Allow admins to set a specific language"?

Reply:
I've rewrote that block without comments and with fewer lines.